packages/e2e-test-utils/src/visitBlockWidgetScreen.js

```diff
@@ -24,20 +24,34 @@ import visitAdminPage from './visitAdminPage';
  * @return {Promise<void>}
  */
 async function visitBlockWidgetScreen() {
+  const WPVersion = process.env?.WP_VERSION;
   await visitAdminPage('widgets.php');
 
   // Disable welcome guide if it is enabled.
-  const isWelcomeGuideActive = await page.evaluate(() =>
-    wp.data
-      .select('core/edit-widgets')
-      .__unstableIsFeatureActive('welcomeGuide')
-  );
+  const isWelcomeGuideActive = await page.evaluate((version) => {
+    // TODO Change after 5.9 release.
+    if ('latest' === version) {
+      return wp.data
+        .select('core/edit-widgets')
+        .__unstableIsFeatureActive('welcomeGuide');
+    }
+    return wp.data
+      .select('core/interface')
+      .isFeatureActive('core/edit-widgets', 'welcomeGuide');
+  }, WPVersion);
```

Comment:
Is the version check really needed? Didn't it work the way you had it before, by checking whether the functions exist?

Reply:
Sadly it didn't work the way I had it before. See https://github.com/google/web-stories-wp/runs/4728644759?check_suite_focus=true#step:11:11

Reply:
I would love another way around doing this check.

Reply:
Hmm sounds like all that is needed is some conditional check, perhaps easiest with optional chaining (`?.`) ,to avoid the `isFeatureActive is not a function` error.

```js
// Disable welcome guide if it is enabled.

// The former selector is for WP < 5.9
const isWelcomeGuideActive = await page.evaluate(() => {
  return wp.data.select('core/edit-widgets')?.__unstableIsFeatureActive('welcomeGuide') ||
  wp.data.select( 'core/interface' )?.isFeatureActive( 'core/edit-widgets', 'welcomeGuide' );
});

if (isWelcomeGuideActive) {
  // The former action is for WP < 5.9
  await page.evaluate(() => {
    wp.data.dispatch('core/edit-widgets')?.__unstableToggleFeature('welcomeGuide');
    wp.data.dispatch( 'core/interface' )?.toggleFeature( 'core/edit-widgets', 'welcomeGuide' );
  });
}
```